tools/issue-labeler/src/Azure.Sdk.Labels/AzureSdkCombinedLabelModelTrainer.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using CreateMikLabelModel;
+using CreateMikLabelModel.ML;
+
+namespace Azure.Sdk.LabelTrainer
+{
+    /// <summary>
+    ///   Provides functionality related to training label models, including building and curating the
+    ///   sets of data needed to do so.
+    /// </summary>
+    ///
+    public  class AzureSdkCombinedLabelModelTrainer : LabelModelTrainer
+    {
+        /// <summary>The set of core Azure SDK language repositories which should be used for training the combined model.</summary>
+        private static readonly string[] AzureSdkLanguageRepositories = new[]
+        {
+            "Azure/azure-sdk-for-net",
+            "Azure/azure-sdk-for-java",
+            "Azure/azure-sdk-for-python",
+            "Azure/azure-sdk-for-js",
+            "Azure/azure-sdk-for-go",
+            "Azure/azure-sdk-for-cpp",
```

Comment:
```suggestion
            "Azure/azure-sdk-for-cpp",
            "Azure/azure-sdk-for-rust",
```

May as well add Rust while we're here.